game/scripts/npc/abilities/crystal_maiden_crystal_nova.txt

```diff
@@ -30,7 +30,7 @@
 
     // Cost
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityManaCost"                                     "130 140 150 160 200 240"
+    "AbilityManaCost"                                     "130 140 150 170 200 240" //OAA
```

Comment:
you made it 5 mana different at max level..... just match dota here....

Reply:
ah come on Chris, nobody plays CM in OAA

Reply:
I see, its 5 mana difference, nvm